arch/aarch64/arch-elf.hh

```diff
@@ -23,4 +23,31 @@ enum {
 
 #define ELF_KERNEL_MACHINE_TYPE 183
 
+#define SAFETY_BUFFER 256
+#include <osv/align.hh>
+
+inline void elf_entry_point(void* ep, int argc, char** argv, int argv_size)
```

Comment:
Maybe we should call it run_entry_point? or jump_to_entry_point? Or something? The name "elf_entry_point" makes it sound like it calculates the entry point - which it doesn't - it takes one.

Reply:
Good idea!

Reply:
Addressed in the updated PR.